### ISSUE TITLE
In the new pK games, why the version 18 have different openings?

### ISSUE THREAD

**Issue by cjohnchen** [leela-zero/leela-zero#2315]
Why only the version 18 have different openings in the PK games? Is there any problem?

**Comment by alreadydone** [comment]
Probably randomness through multithreaded batching, or does LCB lead to more diversity?

**Comment by l1t1** [comment]
did self play game or match have new opning

**Comment by gcp** [comment]
>Probably randomness through multithreaded batching, or does LCB lead to more diversity?

Probably both. Batching will increase exploration a little, and LCB can pick up new 'promising' moves much faster.

**Comment by iopq** [comment]
LCB will pick new openings immediately

![image](https://user-images.githubusercontent.com/1817528/55579194-a6453900-5752-11e9-8a14-c9379ff1572e.png)

fourth move and it already wants to select an alternative

**Comment by Atarust** [comment]
In the screenshot, shouldn't Leela show the other 3-4 move as well, due to symmetry?

**Comment by jokkebk** [comment]
What is pK? LCB probably refers to a recent patch that has the max_lcb_root experiment included?

**Comment by Strappa71** [comment]
@Atarust 
No. It is not symmetric. The shown move is parallel to the black stones. Other 3-4 is perpendicular.
